home.html

```diff
@@ -1,28 +1,30 @@
 <!DOCTYPE html>
```

Comment:
add .DS_STORE and .DS_Store to .gitignore file and then delete it and make sure to not track it.